### PR TITLE
correction in procedure

### PR DESCRIPTION
**Pull request by joinimran**
Added a correction while typing your tenant id during setup.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1753

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/usage-analytics/enable-usage-analytics.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ To initiate the template app, you have to be either a **global administrator**,
 
 5.	Click **Explore app** to view the app with sample data. Click **Connect** to connect the app to your organization’s data.
 
-6.	After clicking **Connect**, on the **Connect to Microsoft 365 usage analytics** screen, type in the tenant Id you copied in step (1) \> **Next**.
+6.	After clicking **Connect**, on the **Connect to Microsoft 365 usage analytics** screen, type in the tenant Id (without dashes) you copied in step (1) **Next**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
6.	After clicking **Connect**, on the **Connect to Microsoft 365 usage analytics** screen, type in the tenant Id (without dashes) you copied in step (1), and select **Next**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #1753.

Thank you.

cc: @kenwith

**Pull request merged**